.github/workflows/build-test.yml

```diff
@@ -547,8 +547,22 @@ jobs:
     - name: Cargo doc
       run: cargo make ci-job-doc
 
+  semver:
+    runs-on: ubuntu-latest
+    steps:
+    # Checkout baseline revision; required because actions/checkout is shallow
+    - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+      with:
+        ref: d38a3efe1ea5b9e9348b5b4b3804bb9ec2017001
```

Comment:
nit: this should perhaps be defined as a variable above

issue: let's document a short policy here on this that describes when the hash can be bumped, and how


can be done in a followup